masq_lib/src/constants.rs

```diff
@@ -3,7 +3,7 @@
 pub const HIGHEST_RANDOM_CLANDESTINE_PORT: u16 = 9999;
 pub const HTTP_PORT: u16 = 80;
 pub const TLS_PORT: u16 = 443;
-pub const DEFAULT_CHAIN_NAME: &str = "mainnet";
+pub const DEFAULT_CHAIN_NAME: &str = "mainnet"; //TODO look, this is different here and in the node's crate; there we call a default 'ropsten' which is quite concerning anyway
```

Comment:
I don't understand this comment; DEFAULT_CHAIN_NAME occurs only once -- right here -- in the whole codebase and it only has "mainnet" -- however TEST_DEFAULT_CHAIN_NAME does exist and it is initialized with "ropsten"

Reply:
The problem is not with DEFAULT_CHAIN_NAME, this works correct, it points to 'mainnet'. Troubles start with DEFAULT_CHAIN_ID defined in utils.rs, even though it's placed in the test_utils directory, every time you see this in the code it can confuse you. Because it seems to be symmetrical to DEFAULT_CHAIN_NAME which it isn't, both point differently, the second to 'ropsten', its index respectively.

So I think I'll settle this with proper names which I'm going to set up with GH-473.